user-manual-for-me/user-manual-for-emily.md

```diff
@@ -0,0 +1,58 @@
+# :zap: User Manual For Emily
+
+_headings marked with a * are optional_
+
+### My time zone
+
+US Pacific (Seattle/LA)
+
+
+### The best way to communicate with me
+1. slack
+1. email (eyspahn@gmail.com)
+1. github
+
+### Tickets or projects that I'm interested in helping out with
+
+Blogging / researching
+
+
+### Languages I speak
+
+|Language | Level |
+| -- | -- 
+|English | Native |
+|Japanese | (it's complicated, let's just call it Beginner)|
+
+
+### Conditions I like to work in *
+Pretty flexible
+
+
+### The best way to receive feedback *
+
+1. slack private message
+1. github comment
+1. slack group
+1. email
+1. video call
+
+### Things I'd like to gain from working on PyLadies Project Communications *
+
+Help others learn and grow. Learning python was fantastic for my career, I'd love to pay i forward.
+
+
+### Times / hours I'd like to work on PyLadies Project Communications activities *
+
+Flexible
+
+
+### Things I love *
+
+dogs, playing music, data nerdery
+
+### Other things to know about me *
+
+civil/environmental engineer turned data scientist and developer
+long time vegan
```

Comment:
For markdown, new lines need to be separated by a blank space or they appear on the same line - gets me all the time